content/tutorials/set-count.md

```diff
@@ -0,0 +1,98 @@
+---
+authors:
+  - name: MaSera
+    title: Contributor
+    url: https://github.com/Maseraphina
+    image_url: https://github.com/Maseraphina.png
+tags: [set count, multiple counting channels]
+---
+import {
+  DiscordActionRow as ActionRow,
+  DiscordAttachments as Attachments,
+  DiscordBold as Bold,
+  DiscordButton as Button,
+  DiscordCommand as Command,
+  DiscordEmbedDescription as Description,
+  DiscordMessages as Discord,
+  DiscordEmbed as Embed,
+  DiscordMessage as Message,
+  DiscordMention as Mention,
+} from "@skyra/discord-components-react";
+
+# How do I change the count?
+This tutorial will give you more information on how to change a count in your server.<br/>(regardless of whether you have 1 or more counting channels in your server)
```

Comment:
don't have `<br/>` in here, either use two lines or just use the same line. no need for a break
alternatively, change the line to `This tutorial will give you more information on how to change a count in your server, regardless of whether you have one or more counting channels in your server.`